### PR TITLE
Adding an integration test to validate JsonStreamFormatter error while sending GET request with messageType property

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Adding the test for the fix for [1] which is done by [2].

## Goals
Add a new integration test to validate JsonStreamFormatter error while sending GET request with messageType property.

## Approach
After writing the integration test, I have run all the integration tests locally as shown below and all are passed.
image

[1] https://github.com/wso2/product-ei/issues/5348
[2] https://github.com/wso2-support/wso2-synapse/pull/1055

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        wireServer = new WireMonitorServer(8991);
+        AuthenticatorClient login = new AuthenticatorClient(gatewayContextMgt.getContextUrls().getBackEndUrl());
+        String session = login.login("admin", "admin", "localhost");
+        // Upload the synapse
+        String file = "artifacts" + File.separator + "AM" + File.separator + "synapseconfigs" + File.separator
+                + "property" + File.separator + "dummy_api_with_in_sequence.xml";
+        OMElement synapseConfig = APIMTestCaseUtils.loadResource(file);
+        APIMTestCaseUtils.updateSynapseConfiguration(synapseConfig, gatewayContextMgt.getContextUrls().getBackEndUrl(),
+                session);
+        Thread.sleep(5000);
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java:57]: Why do we have the sleep here?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+public class APIInvocationWithMessageTypeProperty extends APIMIntegrationBaseTest {
+
+    private WireMonitorServer wireServer;
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init();
+        wireServer = new WireMonitorServer(8991);
+        AuthenticatorClient login = new AuthenticatorClient(gatewayContextMgt.getContextUrls().getBackEndUrl());
+        String session = login.login("admin", "admin", "localhost");
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java:50]: Can we use constants for these?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java (hunk 1 above)
Comment (wasuradananjith) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java]: Removed via https://github.com/wso2/product-apim/pull/9915/commits/dc0c03d044f063e3b680ee248804d27665fd2217

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java (hunk 2 above)
Comment (wasuradananjith) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIInvocationWithMessageTypeProperty.java]: Fixed via https://github.com/wso2/product-apim/pull/9915/commits/29fe4189618d0cb1af533576fd790f1f485cf46e

**Review by mushthaq33 — APPROVED** [review]

**Pull request merged**